### PR TITLE
Put frontend on the car

### PR DESCRIPTION
**Pull request by BANANAPEEL202**

### CODE REVIEW

File: .DS_Store
Comment (RCMast3r) [inline .DS_Store:1]: remove this file

File: dbc_proto_gen_script.nix
Comment (RCMast3r) [inline dbc_proto_gen_script.nix:1]: - [ ] remove this change here

File: default.nix
Comment (RCMast3r) [inline default.nix:1]: - [ ] remove this too (override python attrs that is)

File: fix_mcaps.py
Comment (RCMast3r) [inline fix_mcaps.py:1]: i believe this script can be removed

File: frontend/.idea/.gitignore
Comment (RCMast3r) [inline frontend/.idea/.gitignore:1]: - [ ] remove and blacklist the .idea folder

File: frontend_webserver/.idea/.gitignore
Comment (RCMast3r) [inline frontend_webserver/.idea/.gitignore:1]: - [ ] remove and blacklist the frontend_webserver .idea file too ....

File: py_data_acq/.DS_Store
Comment (RCMast3r) [inline py_data_acq/.DS_Store:1]: - [ ] remove yet another .DS_Store file ...

File: py_data_acq/py_data_acq/web_server/mcap_server.py
Comment (RCMast3r) [inline py_data_acq/py_data_acq/web_server/mcap_server.py:1]: are we sure that this has to be 192.168.203.1?

**Review by PKuppili — COMMENTED** [review]
Works! Has some unnecessary files/scripts that we'll remove before merge but nice work overall!

**Review by PKuppili — APPROVED** [review]

File: py_data_acq/py_data_acq/web_server/mcap_server.py
Comment (BANANAPEEL202) [reply py_data_acq/py_data_acq/web_server/mcap_server.py:1]: iirc it doesn't work on 0.0.0.0, which doesn't make sense to me

File: fix_mcaps.py
Comment (BANANAPEEL202) [reply fix_mcaps.py:1]: I don't think we use this file either.

**Pull request merged**